### DISCUSSION TITLE
New Community Working Group chairs elected

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/discussions#110] [Community Working Group]
We are delighted to announce that the following members of the Green Software Foundation have been elected as Chairs of the Community Working Group for the next 2 years:

* Anne Currie - re:cinq - @aecurrie
* Gadhu Sundaram - NTT Data - @GadhuNTTDATA
* Pindy Bhullar - UBS - @PindyBhullar

I would like you to join me in congratulating them on their appointment! 🎉

**Comment by aecurrie** [comment]
Welcome aboard Gadhu! Lovely to have you Pindy!!!!

**Comment by GadhuNTTDATA** [comment]
Thanks Anne!! Looking forward to working with both of you!!
